src/Runners/PHPUnit/Options.php

```diff
@@ -115,6 +156,18 @@ public function __get($var)
     }
 
     /**
+     * Public read accessibility
+     * (e.g. to make empty($options->property) work as expected)
+     *
+     * @param string $var
+     * @return mixed
+     */
+    public function __isset($var)
```

Comment:
They're protected inside the class. __get and __isset are created to add **read-only** access outside the class. __set and __unset deliberately aren't